### DISCUSSION TITLE
Pods not resolving several host

### DISCUSSION THREAD

**Discussion by jamorales85** [okd-project/okd#958] [General]
**Describe the bug**
Pods don't resolve several external links.
```
argocd-server-7cd5794cc5-khh8q:/tmp$ git clone https://github.com/argoproj/argocd-example-apps.git
Cloning into 'argocd-example-apps'...
fatal: unable to access 'https://github.com/argoproj/argocd-example-apps.git/': Could not resolve host: github.com
```

**Version**
`4.8.0-0.okd-2021-10-24-061736`

**How reproducible**
```
argocd-server-7cd5794cc5-khh8q:/tmp$ git clone https://github.com/argoproj/argocd-example-apps.git
Cloning into 'argocd-example-apps'...
fatal: unable to access 'https://github.com/argoproj/argocd-example-apps.git/': Could not resolve host: github.com
```
**Notes**
Workaround: Pods node-resolver-* from openshift-dns namespace has the next error:
```
[root@okd4-compute-1 /]# host www.github.com
host: parse of /etc/resolv.conf failed
[root@okd4-compute-1 /]# cat /etc/resolv.conf 
search 
nameserver mydns.com
```
If I fix the /etc/resolv.conf in those pods, the cluster works properly.

**Comment by vrutkovs** [comment]
Please attach (or upload to the public file sharing service) [must-gather](https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) archive

**Reply by jamorales85** [reply]
I have a file do you need?
